ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -144,33 +150,38 @@ def prepare_lookahead_waypoints(self):
             end_pos = closest_wp_pos + LOOKAHEAD_WPS - 1  # to build waypoint list with a fixed size
             next_waypoints = list(islice(seq, closest_wp_pos, end_pos))  # list of lookahead waypoints
 
-            max_vel = self.mph_to_mps(50)
             if self.traffic is None or self.traffic == -1:
```

Comment:
I dont think we need the condition self.traffic is None. I looked into tl_detector.py- either light_wp or self.last_wp does not have None value. So there is no chance that we will come across this condition self.traffic is None. Only self.traffic == -1 is enough here I think.